docs/benchmarking-tool-overview.md

```diff
@@ -139,48 +156,48 @@ Following the grafana dashboard structure, which wraps all the metrics currently
         * same as for the aforementioned, but from the Pool’s perspective
 * **SV1 roles performances**
     * <span style="text-decoration:underline;">Pool roles - CPU usage</span>
-        * _Description_: it tracks the CPU usage (percentage) due to SV1 roles running as Pool’s infrastructure (SV1 Pool and SV1 Bitcoin node)
+        * _Description_: CPU usage (percentage) of SV1 roles running as Pool’s infrastructure (SV1 Pool and SV1 Bitcoin node)
         * _Data Collection Method_: [cadvisor](https://github.com/google/cadvisor) is used to constantly track the CPU usage of every Docker container 
     * <span style="text-decoration:underline;">Pool roles - memory usage</span>
-        * _Description_: it tracks the memory usage due to SV1 roles running as Pool’s infrastructure (SV1 Pool and SV1 Bitcoin node)
+        * _Description_: Memory usage of SV1 roles running as Pool’s infrastructure (SV1 Pool and SV1 Bitcoin node)
         * _Data Collection Method_: [cadvisor](https://github.com/google/cadvisor) is used to constantly track the memory usage of every Docker container 
     * <span style="text-decoration:underline;">Pool roles - Network Tx</span>
-        * _Description_: it tracks the amount of data transmitted from the  SV1 roles running as Pool’s infrastructure (SV1 Pool and SV1 Bitcoin node)
+        * _Description_: Amount of data transmitted from the SV1 roles running as Pool’s infrastructure (SV1 Pool and SV1 Bitcoin node)
         * _Data Collection Method_: [cadvisor](https://github.com/google/cadvisor) is used to constantly track the bandwidth usage of every Docker container 
-        * _Data Analysis and usage_: since cadvisor tracks metrics on a container level, the total amount of bandwidth measured even includes the one consumed between SV1 Pool and Bitcoin node, which are typically ran in the same network, so it’s not meaningful
+        * _Data_: since cadvisor tracks metrics on a container level, the total amount of bandwidth measured even includes the one consumed between SV1 Pool and Bitcoin node, which are typically ran in the same network, so it’s not meaningful
```

Comment:
Why do we have the metric if it's not meaningful? Same for others labeled in the same way.